libs/cardano-ledger-api/src/Cardano/Ledger/Api/State/Query.hs

```diff
@@ -123,16 +111,6 @@ import qualified Data.Set as Set
 import Lens.Micro
 import Lens.Micro.Extras (view)
 
--- | Filter out stake pool delegations and rewards for a set of stake credentials
-filterStakePoolDelegsAndRewards ::
```

Comment:
Shouldn't we replace this with a version that takes `Accounts` instead of `UMap`?  Or it's too trivial to have at all in the api?